src/components/Messages/PreviewList.tsx

```diff
@@ -7,19 +7,22 @@ import { GridItemFour } from '@components/UI/GridLayout';
 import { Modal } from '@components/UI/Modal';
 import { PageLoading } from '@components/UI/PageLoading';
 import useMessagePreviews from '@components/utils/hooks/useMessagePreviews';
+import useWindowSize from '@components/utils/hooks/useWindowSize';
 import type { Profile } from '@generated/types';
 import { MailIcon, PlusCircleIcon } from '@heroicons/react/outline';
 import buildConversationId from '@lib/buildConversationId';
 import { buildConversationKey } from '@lib/conversationKey';
 import isFeatureEnabled from '@lib/isFeatureEnabled';
 import { useRouter } from 'next/router';
 import type { FC } from 'react';
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import Custom404 from 'src/pages/404';
 import Custom500 from 'src/pages/500';
 import { useAppStore } from 'src/store/app';
 import { useMessageStore } from 'src/store/message';
 
+const MIN_WIDTH_FOR_TWO_PANELS = 1024;
```

Comment:
I think the mobile flow starts 768px, we should have that here

Reply:
So, maybe it should start at 768. RIght now, between 768 and 1024 it doesn't hide the conversation list when you have a conversation loaded. I did make a separate ticket for that. https://github.com/xmtp-labs/hq/issues/820

Reply:
Also move the constant to the constants file

Reply:
Ya, that's a good call

Reply:
Okay so are you suggesting we change to a mobile view below 1024px?

I think that will make sense.